Dockerfile.dev

```diff
@@ -0,0 +1,28 @@
+FROM python:3.7.5-stretch
```

Comment:
Why is the existing Dockerfile not sufficient?

Reply:
I get this error:

```
Could not find 'mono' host. Make sure that 'mono' is installed on the machine and is available in PATH environment variable.

Test Run Aborted.
```

I do have `mono` version `6.12.0.122` on the host machine.

Reply:
Updated the `CONTRIBUTING.md` guidance with some additional gotchas and solutions. However, I could not get the existing `Dockerfile` to work locally, only the new `Dockerfile.dev`